integration/test_timescaledb.py

```diff
@@ -138,36 +121,38 @@ def timescaledb(request: "pytest.FixtureRequest") -> Iterator[TimescaleDBData]:
         "timestamp_col",
         "nchar_col",
         "varchar_col",
+        "text_col",
     ]
 
-    yield (
-        test_connection,
-        conn_host,
-        conn_port,
-        conn_database,
-        conn_user,
-        conn_password,
+    return TimescaleDBData(
         table_name,
         dsn,
         timestamp_precision,
         columns_config,
     )
 
-    # Cleanup
-    try:
-        cleanup_cursor = test_connection.cursor()
-        cleanup_cursor.execute(f"DROP TABLE IF EXISTS {table_name} CASCADE;")
-        cleanup_cursor.close()
-    except Exception as e:
-        print(f"Cleanup error: {e}")
-    test_connection.close()
 
+@pytest.fixture(params=[("milliseconds"), ("microseconds")])
```

Comment:
No-op parentheses. Either this should be:
```suggestion
@pytest.fixture(params=["milliseconds", "microseconds"])
```
or there is a bug here, and it should be
```suggestion
@pytest.fixture(params=[("milliseconds",), ("microseconds",)])
```

Reply:
will fix